oneview/provider.go

```diff
@@ -69,6 +69,7 @@ func Provider() terraform.ResourceProvider {
 
 		DataSourcesMap: map[string]*schema.Resource{
 			"oneview_appliance_snmp_v1_trap_destinations": dataSourceSNMPv1TrapDestination(),
+			"oneview_snmpv3_trap_destinations":            dataSourceSNMPv3TrapDestination(),
```

Comment:
Can you change to oneview_appliance_snmpv3_trap_destinations or change the resource order.